src/renderer/viz/expressions.js

```diff
@@ -97,20 +108,26 @@
  * ```
  *
  * @typedef {} Color
+ * @name Color
+ * @memberof Types
  * @api
  */
 
 /**
  * Type of Date Expressions.
  *
  * @typedef {} Date
+ * @name Date
+ * @memberof Types
```

Comment:
I don't understand why the types are "memberof" `Types`. There is no constructor on `carto.Types. I find this confussing.

Reply:
🔝